src/dune_rules/simple_rules.ml

```diff
@@ -192,9 +192,13 @@ let copy_files sctx ~dir ~expander ~src_dir (def : Copy_files.t) =
         let file_dst = Path.Build.relative dir basename in
         Path.build file_dst)
   in
-  Option.iter def.alias ~f:(fun alias ->
+  let+ () =
```

Comment:
Another use case for `Memo.Option.iter`.

Reply:
updated